### PR TITLE
4zK's content update #5: Finishing up for Prerelease 3

### PR DESCRIPTION
**Pull request by fourZK**
Just a lot of fine tunings. Cleared up a lot of unneeded and uninformative .ini definitions that correspond to defaults/nothing.

### CODE REVIEW

File: Base.rte/Devices/Tools/GrappleGun/GrappleGun.ini [hunk 1]
```diff
@@ -147,16 +128,11 @@ AddAmmo = Magazine
 	DeepCheck = 0
 	JointStrength = 200
 	JointStiffness = 1
-	JointOffset = Vector
```
Comment (MaximDude) [inline Base.rte/Devices/Tools/GrappleGun/GrappleGun.ini:150]: Keep `JointOffset = 0/0` on non-null sprite presets.

File: Browncoats.rte/Actors/Infantry/BrowncoatHeavy/BrowncoatHeavy.ini [hunk 2]
```diff
@@ -98,15 +98,19 @@ AddEffect = Attachable
 	DeepCheck = 0
 	JointStrength = 50
 	JointStiffness = 0.6
-	JointOffset = Vector
```
Comment (MaximDude) [inline Browncoats.rte/Actors/Infantry/BrowncoatHeavy/BrowncoatHeavy.ini:101]: Keep `JointOffset`.

File: Browncoats.rte/Devices/Weapons/Longshot/Longshot.ini [hunk 3]
```diff
@@ -65,18 +65,13 @@ AddAmmo = Magazine
 	DeepCheck = 0
 	JointStrength = 200
 	JointStiffness = 1
-	JointOffset = Vector
```
Comment (MaximDude) [inline Browncoats.rte/Devices/Weapons/Longshot/Longshot.ini:68]: Keep `JointOffset`.

File: Browncoats.rte/Devices/Weapons/Stinger/Stinger.ini [hunk 4]
```diff
@@ -67,18 +67,13 @@ AddAmmo = Magazine
 	DeepCheck = 0
 	JointStrength = 200
 	JointStiffness = 1
-	JointOffset = Vector
```
Comment (MaximDude) [inline Browncoats.rte/Devices/Weapons/Stinger/Stinger.ini:70]: Keep `JointOffset`.

File: Imperatus.rte/Devices/Weapons/Mauler/Mauler.ini [hunk 5]
```diff
@@ -173,18 +167,13 @@ AddAmmo = Magazine
 	DeepCheck = 0
 	JointStrength = 200
 	JointStiffness = 1
-	JointOffset = Vector
```
Comment (MaximDude) [inline Imperatus.rte/Devices/Weapons/Mauler/Mauler.ini:176]: Keep `JointOffset`.

File: Ronin.rte/Devices/Weapons/M1Garand/M1Garand.ini [hunk 6]
```diff
@@ -62,18 +62,13 @@ AddAmmo = Magazine
 	DeepCheck = 0
 	JointStrength = 200
 	JointStiffness = 1
-	JointOffset = Vector
```
Comment (MaximDude) [inline Ronin.rte/Devices/Weapons/M1Garand/M1Garand.ini:65]: Keep `JointOffset`.

**Review by MaximDude — CHANGES_REQUESTED** [review]

File: Base.rte/Scripts/Shared/SmokeTrail.lua [hunk 7]
```diff
@@ -1,18 +1,30 @@
...
-			effect.Vel = self.Vel * RangeRand(0.75, 1.0);
-			effect.Lifetime = self.smokeTrailLifeTime * RangeRand(1.0, 1.5);
+			effect.Vel = self.Vel * RangeRand(0.75, 1);
+			effect.Lifetime = self.smokeTrailLifeTime * RangeRand(0.5, 1);
+		
+			if self.smokeTrailTwirl > 0 then
+				effect.AirResistance = effect.AirResistance * RangeRand(0.9, 1);
+				effect.GlobalAccScalar = effect.GlobalAccScalar * math.random();
+
+				effect.Pos = self.Pos - offset + offset * i/trailLength;
```
Comment (garethyr) [inline Base.rte/Scripts/Shared/SmokeTrail.lua:23]: In cases like these I like to put some brackets around the multiplication, just to make it entirely obvious what the order of operations is. I find it improves readability.

**Review by garethyr — CHANGES_REQUESTED** [review]
One super minor change for ya.

**Review by MaximDude — APPROVED** [review]

**Review by garethyr — APPROVED** [review]

**Pull request merged**